packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -855,6 +857,11 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
 
   protected getActiveDescendant() {
     const currentIndex = this.suggestionStore.currentIndex;
+
+    if (currentIndex < 0 && this.suggestionElement.current && this.suggestionElement.current.hasSuggestedAction()) {
```

Comment:
I think that there is a subtle bug, if  `this.state.suggestionsLoading` then `currentIndex < 0` will be true, because there are no suggestions. However, `hasSuggestedAction` will evaluate to true because the buttons are present in the dom. It's not a huge problem because the activeDescendant will point to a valid dom object, but it means that the buttons might get read out when it shouldn't.

It looks like if `this.suggestionsLoading` evaluates to true, then we could return undefined immediately.

Reply:
@joschect that makes sense to me! I'll add another `if` block at the top.

Another outside option is if the loading text were given an `id`, `activedescendant` could point to it when loading rather than use a live region to notify screen readers

I don't want to expand the scope of this PR too much, so I'll add `undefined` for now, but if you like that idea it could be a nice change in the future 😃